src/Components/Modal/modal.js

```diff
@@ -1,24 +1,100 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./modal.css";
 import ModalItems from "./Modal-Items/ModalItems";
 
 const Modal = () => {
-	return (
-		<div className="modal-class">
-			<p>Available Routes</p>
-			<span>
-				<label>Direction</label>
-				<label>Duration</label>
-				<label>Vehicle</label>
-				<label>Cost</label>
-			</span>
-			<ModalItems />
-			<ModalItems />
-			<ModalItems />
-			<ModalItems />
-			<ModalItems />
-		</div>
-	);
+	const [routes, setRoutes] = useState([]);
+
+	const fetchRoutes = () => {
```

Comment:
please put this in a separate file and import it here. it doesn't have to be here